src/main/java/control/ControllerManager.java

```diff
@@ -146,7 +148,7 @@ private void handleCancel(MouseEvent event) {
      *
      * @param block ShotBlock to set as active
      */
-    public void setActiveShotBlock(ShotBlock block) {
+    public void setActiveShotBlock(Optional<ShotBlock> block) {
```

Comment:
I would change the set to an actual `ShotBlock`, you can still put in a `null` variable if I am correct. Why would I want to set it to something if there is a possibility there is nothing in the variable I am setting it with?